test.js

```diff
@@ -58,3 +59,15 @@ test('pause with handler', async (t) => {
 
   session.destroy()
 })
+
+test('heap snapshot', async (t) => {
+  const session = new inspector.Session()
+
+  session.connect()
+
+  const snapshot = new HeapSnapshot(session)
+
+  for await (const chunk of snapshot) {
+    t.ok(chunk)
```

Comment:
There's some risk that this test ends up not testing anything, because it would pass if it yields no chunks whatsoever. Maybe test for that explicitly?

```
  let chunks = 0
  for await (const chunk of snapshot) {
    t.ok(chunk)
    chunks++
  }
  t.ok(chunks > 0)
```

Reply:
The only thing it tests is that the thing doesn't blow up. I'll add another assertion though, seems like an improvement nonetheless!